In [7]:
import math as M

### Example 3.1
Copper, a metal known since ancient times, is used in electrical cables and pennies, among other things.  

The atomic masses of its two stable isotopes, $_{63}^{29}Cu$ (69.09 percent) and $_{65}^{29}Cu$ (30.91 percent), are 62.93 amu and 64.9278 amu, respectively.

Calculate the average atomic mass of copper. The relative abundances are given in parentheses.



In [2]:
(62.93*0.6909)+(64.9278*0.3091)

63.54751998

# 3.2 Avogadro’s Number and the Molar Mass of an Element

<img src="Screens/figure3_2.jpg">

In [30]:
AVOGADRO = 6.0221415 * M.pow(10,23)
AMU_IN_GRAMS = 1.661 * M.pow(10,-24)

###### Hvordan tenke på mol
En mol med grunnstoff veier amu oppgitt i periodetabellen

### Example 3.2
Zinc (Zn) is a silvery metal that is used in making brass (with copper) and in plating iron to prevent corrosion. How many moles of Zn are there in 45.9 g of Zn?

In [5]:
Zn_amu = 65.38
m = 45.9

n = m / Zn_amu
n

0.7020495564392781

### Example 3.3
Sulfur (S) is a nonmetallic element that is present in coal. When coal is burned, sulfur is converted to sulfur dioxide and eventually to sulfuric acid, which gives rise to the acid rain phenomenon. How many atoms are in 25.1 g of S?

In [42]:
S_amu = 32.06
m = 25.1
n = m /S_amu
N = n * AVOGADRO
print(round(N/(M.pow(10,23)),2),"e23 mol")

4.71 e23 mol


###### Alternativ metode for 3.3
Løser ved hjelp av conversion factors

Sulfur amu: $32.06 mol/g$

$\frac{32.06 \frac{mol}{g}}{25g} = \frac{25g mol}{32.06g} = 0.78... mol = 4.71  \text{ E23 atoms}$

Kan også skrive det enda enklere:

$\text{n atoms } = \frac{32}{25} \cdot avogadro$

In [46]:
n = m  * 1/S_amu 
print(n)
N =  n * AVOGADRO
N

0.7829070492825951


4.714777032127262e+23

In [48]:
25/32

0.78125

### Example 3.4
Silver (Ag) is a precious metal used mainly in jewelry. What is the mass (in grams) of one Ag atom?

In [41]:
Ag_amu = 107.87
N = 1
n = N / AVOGADRO
m = n * Ag_amu
m
print(round(m*(M.pow(10,22)),3),"E-22 g")

1.791 E-22 g


###### Alternativ metode for 3.4
Løser ved hjelp av conversion factors

In [32]:
Ag_amu * AMU_IN_GRAMS

1.7917207e-22